src/containers/App/App.scss

```diff
@@ -119,8 +119,7 @@ body,
 .data-table__row,
 .ydb-virtual-table__row,
 .ydb-tree-view__item {
-    &:hover,
-    &:focus-within {
+    &:hover {
```

Comment:
let's add `focus-visible` here, to make the button visible if iterating with keyboard.

Reply:
Elements with `visibility:hidden` cannot be selected focused with tab, so it doesn't have any effect

Reply:
100%! What do you think if we hide them with `opacity=0`?

Reply:
`opacity=0` works just fine, updated PR